src/start_menu/main_settings/size.cpp

```diff
@@ -0,0 +1,9 @@
+#include "size.h"
+#include "../../utils/utils.h"
+#include <iostream>
+
+void Size(){
+	std::cout << "Pick Number" << std::endl;
```

Comment:
Perhaps some more words here to inform the user of what that 'number' represents?

Reply:
done